File: javascript/postgres-js/test/alternatives/no_connection_pool/example_with_no_connection_pool.test.js
/*
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */

import { example } from "../../../src/alternatives/no_connection_pool/example_with_no_connection_pool.js";

test("Example with no connection pool", async () => {
  await example();
}, 20000);

File: java/pgjdbc/src/test/java/software/amazon/dsql/examples/alternatives/no_connection_pool/ExampleWithNoConnectionPoolTest.java
/*
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */

package software.amazon.dsql.examples.alternatives.no_connection_pool;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;

public class ExampleWithNoConnectionPoolTest {
    @Test
    public void testExampleWithNoConnectionPool() {
        assertAll(() -> ExampleWithNoConnectionPool.main(new String[]{}));
    }
}
